src/main/java/Mario/Tasks/ToDo.java

```diff
@@ -0,0 +1,32 @@
+package Mario.Tasks;
+
+import Mario.Tasks.Task;
+
+public class ToDo extends Task {
```

Comment:
Please consider renaming this class to `Todo` so that it follows the coding standards